editor/project_manager.cpp

```diff
@@ -2554,6 +2554,7 @@ ProjectManager::ProjectManager() {
 
 	Button *rename = memnew(Button);
 	rename->set_text(TTR("Rename"));
+	// The F2 shortcut isn't overridden with Enter on macOS as Enter is already used to edit a project.
```

Comment:
Should there be a different shortcut to edit the project on macOS maybe? What would be idiomatic there? Cmd+E? CC @godotengine/macos

Reply:
https://github.com/godotengine/godot/pull/47894 already added <kbd>Ctrl + E</kbd> / <kbd>Cmd + E</kbd>, but only on `master`.